### PR TITLE
feat: add delete token

### PR DESCRIPTION
**Pull request by feritka79**
add delete token

### CODE REVIEW

File: src/routes/token/deleteToken.ts
Comment (ceydae) [inline src/routes/token/deleteToken.ts:1]: rename to delete.ts

File: src/routes/token/deleteToken.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const deleteToken: RequestHandler = async (req, res) => {
+  try {
+    const { token } = req.body;
+
+    const isTokenExist = await Token.findOne({ address: token });
+    if (!isTokenExist) {
+      return res.status(400).json(
+        responseTemplate({
+          status: 'error',
+          message: 'Token not exists',
```
Comment (ceydae) [inline src/routes/token/deleteToken.ts:16]: change response to Token does not exist in the database

File: src/routes/token/deleteToken.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        }),
+      );
+    }
+
+    const tokenDeleted = await removeToken(token);
+
+    return res.status(200).json(
+      responseTemplate({
+        status: 'success',
+        message: 'Deleted token successful',
```
Comment (ceydae) [inline src/routes/token/deleteToken.ts:27]: Token deleted successfully

File: src/utils/token/removeToken.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  } catch (e) {
+    if (e instanceof Error) {
+      responseTemplate({
+        status: 'error',
+        message: e.message,
+        result: {},
+      });
+    }
+  }
+};
```
Comment (ceydae) [inline src/utils/token/removeToken.ts:21]: move this content to the API file directly and remove this file.

Comment (ceydae) [reply src/utils/token/removeToken.ts:21]: change response to "Token deleted successfully"

**Review by ceydae — CHANGES_REQUESTED** [review]

**Review by feritka79 — COMMENTED** [review]
fix

**Pull request closed without merging**